.github/workflows/ci.yaml

```diff
@@ -64,22 +64,26 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: "1.20"
 
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Download tool dependencies
+        run: make deps
+
+      # Ensure that generated files were committed.
+      - name: Verify
+        run: make verify
+
```

Comment:
Any particular reason to move this up?

Reply:
I did it to avoid running CI scripts if generated code was not committed. In this scenario, we test not the actual code that should be tested but some intermediate state.  I wanted to avoid it as early as possible. If you think it is worth clarifying, I can leave a comment.

Reply:
Makes sense!